bindings/python/test/environment/objects/test_celestial.py

```diff
@@ -0,0 +1,15 @@
+# Apache License 2.0
+
+from ostk.physics.environment.objects.celestial_bodies import Earth
+
+class TestCelestial:
+
+    def test_model_is_defined_success(self):
+        earth = Earth.spherical()
+    
+        assert earth.gravitational_model_is_defined() is True
```

Comment:
```suggestion
        assert earth.gravitational_model_is_defined()
```

Reply:
I think we tend to be explicit in bool comparisons. See here for an example: https://github.com/open-space-collective/open-space-toolkit-physics/blob/main/bindings/python/test/coordinate/test_frame.py